state/model.go

```diff
@@ -459,32 +459,51 @@ func (m *Model) ControllerTag() names.ControllerTag {
 }
 
 // UUID returns the universally unique identifier of the model.
+//
```

Comment:
I appreciate the intent and the effort here but this should have been a seperate PR as not to confuse the intention of this one.

Reply:
happy to put this in a separate PR if you insist, but I understand that generally comment changes are fine as drive-bys.

We're already acting if as these functions are deprecated (we're replacing all usages with the new service methods). The idea with the deprecation warnings was just to document which service methods should be used to replace each state method. For this one it's fairly obvious and I think everyone knows about the model info service. However, for other ones, such the `GetModelUsers` method that @Aflynn50 recently added to replace `Users`, people may have not seen the original PR and they won't know that there is a method available to replace the state `Model.Users` method.

Reply:
Put into a separate PR: #17839